vortex-flatbuffers/flatbuffers/vortex-scalar/scalar.fbs

```diff
@@ -1,12 +0,0 @@
-include "vortex-dtype/dtype.fbs";
-
-table Scalar {
```

Comment:
we never serialized scalar as flatbuffer and it didn't seem worth it to keep ScalarValue table with single field